### ISSUE TITLE
Fix support for encoded URLs

### ISSUE THREAD

**Issue by mems** [remarkjs/remark-validate-links#31]
In my repo I've got paths/files with special chars (spaces, comma, etc.).  
Markdown format use URL for links, which are [encoded](https://en.wikipedia.org/wiki/Percent-encoding) like:
```markdown
[A Link](./Path%20to/File.md)
```

But when I use remark-validate-links, I've got the warning:

```
warning  Link to unknown file: `Path%20to/File.md`  missing-file  remark-validate-links
```

---

If I try to decode URLs (`[A Link](./Path to/File.md)`), remark parse it as a non link.  
That's means it's stringified as `[A Link]\(./Path to/File.md)` and validate-links doesn't output any warning.

**Comment by wooorm** [comment]
Good catch! Could you try creating a PR to fix this? I’m thinking of `decodeURI`
